cli/agent.go

```diff
@@ -314,6 +316,14 @@ func (r *RootCmd) workspaceAgent() *serpent.Command {
 				return xerrors.Errorf("create agent execer: %w", err)
 			}
 
+			var containerLister agentcontainers.Lister
+			if !devcontainersEnabled {
+				containerLister = &agentcontainers.NoopLister{}
+			} else {
+				logger.Info(ctx, "agent devcontainer detection not enabled")
```

Comment:
```suggestion
				logger.Info(ctx, "agent devcontainer detection enabled")
```
Should we log in the other case as well?

Reply:
oops! updated.